client.go

```diff
@@ -0,0 +1,92 @@
+package radius
+
+import "sync"
+
+// Clients are list of client allowed to communicate with server
+type Clients struct {
```

Comment:
I think ClientList may be better than Clients.
I used to avoid Plural and Past tense of the word.
It is a little difficult for non-english developers to understand Plural stuff.